policy/score_calculator.go

```diff
@@ -67,28 +67,6 @@ func AddSpecdScore(calculator ScoreCalculator, s *Score, found bool, impact *exp
 		return
 	}
 
-	// everything else is modify or activate
-
-	if impact.Scoring == explorer.ScoringSystem_IGNORE_SCORE {
```

Comment:
in #1350 I already made sure all the calculators handle exceptions properly, so this workaround shouldn't be needed anymore